controls/overlay.rb

```diff
@@ -0,0 +1,26 @@
+# Semantic changes
+
+overlay_controls = input('overlay_controls')
+system_categorization = input('system_categorization')
+
+include_controls 'k8s-cluster-stig-baseline' do
+
+  ## NA due to the requirement not included in CMS ARS 5.0
+  unless overlay_controls.empty?
+    overlay_controls.each do |overlay_control|
+      control overlay_control do
+        impact 0.0
+        desc "caveat", "Not applicable for this CMS ARS 5.0 overlay, since the requirement is not included in CMS ARS 5.0"
+      end
+    end
+  end
+
+  # Non-ARS control
+  control "V-242437" do
+    impact 0.0
+    desc "caveat", "Not applicable for this CMS ARS 5.0 overlay, since the requirement is not included in CMS ARS 5.0"
+    describe "Not applicable for this CMS ARS 5.0 overlay, since the requirement is not included in CMS ARS 5.0" do 
+      skip "Not applicable for this CMS ARS 5.0 overlay, since the requirement is not included in CMS ARS 5.0"
+    end
```

Comment:
Hey @ssayed118, I believe we have been adding Non-ARS controls in `inspec.yml` with a comment like this `# Not in ARS 5.0` for all system categorizations. So, this can be removed from here. Lmk if you have any questions!

Reply:
Correct. In this case, V-242437 should be added to the values of every baseline in inspec.yml